core/src/main/java/nu/yona/server/goals/entities/IGoal.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2020 Stichting Yona Foundation
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v.2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package nu.yona.server.goals.entities;
+
+import nu.yona.server.analysis.entities.DayActivity;
+import nu.yona.server.util.TimeUtil;
+
+import java.time.LocalDateTime;
+import java.time.ZonedDateTime;
+import java.time.temporal.TemporalUnit;
+import java.util.Arrays;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Arrays'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AUselessImportCheck)